src/modules/localStr.js

```diff
@@ -0,0 +1,4 @@
+export const form = document.getElementById('todoform');
+export const todoInput = document.getElementById('newtodo');
+export const todosListEl = document.getElementById('todos-list');
+export const clearBtn = document.getElementById('clear-all-btn');
```

Comment:
You did an amazing job of creating and using ES6 modules but I think it can be better if the module has a specific functionality. Maybe here you can try to some information not only declaring those variables.